website/src/parsers/js/hermes.js

```diff
@@ -0,0 +1,44 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import HermesWorkerClient from './hermes/HermesWorkerClient';
+
+const ID = 'hermes';
+export const defaultOptions = {
+  jsx: true,
+};
+export const parserSettingsConfiguration = {
+  fields: [
+    'jsx',
+  ],
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: 'N/A',
+  homepage: 'https://hermesengine.dev/',
+  locationProps: new Set(['loc', 'range']),
+
+  loadParser(callback) {
+    callback(new HermesWorkerClient());
+  },
+
+  async parse(hermes, code, options) {
+    const hermesOutput = await hermes.run(['-dump-ast', '--pretty-json', '-dump-source-location', options.jsx ? '-parse-jsx' : ''].filter(Boolean), code);
+    try {
+      return JSON.parse(hermesOutput);
+    } catch (e) {
+      console.error(e);
+      throw new Error(hermesOutput);
```

Comment:
Why does this ignore original error object? Which one has actual detailed information?

Reply:
Ah I see. Yeah, looks like currently this integration doesn't provide proper source location for errors:
![image](https://user-images.githubusercontent.com/557590/82708514-54bd7d80-9c76-11ea-9474-3811c14bcbaa.png)